ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh

```diff
@@ -86,12 +128,14 @@ function set_users () {
         source "${SHARED_DIR}/runtime_env"
     fi
     runtime_env=${SHARED_DIR}/runtime_env
```

Comment:
```suggestion
    runtime_env="${SHARED_DIR}/runtime_env"
```

Reply:
I'd like to keep original line that is not introduced by this PR.